tools/spelling_dictionary.txt

```diff
@@ -88,6 +89,7 @@ EWOULDBLOCK
 EXPECTing
 EXPECTs
 EXPR
+FALLTHROUGH
```

Comment:
if you use the builtin macro you can drop this dictionary addition.

Reply:
Done.